### ISSUE TITLE
jsFuncCall not recognized

### ISSUE THREAD

**Issue by anhari** [pangloss/vim-javascript#837]
Not sure what happened, but recently function calls started being recognized as `jsObjectProp`'s instead of `jsFuncCall`'s for me.

**Comment by bounceme** [comment]
going to need a detail or two

**Comment by Reewr** [comment]
I just recently updated to a newer version of this plugin and saw the same issue. Previously, in the example below, the `log` statement would have been coloured as it was a function call. 

Now it seems like function calls that are on objects are not considered for the `jsFuncCall` but has instead moved to `jsObjectProp` if @anhari is correct.

```js

console.log("Foo");
        ^^^
// log is not matched on jsFuncCall

// `foo` is still matched correctly.
foo();
```

This could be an intended change(?) as `log` is technically a `jsObjectProp`, but it's also a `jsFuncCall`. Personally, I'd like the latter to take precedence as I prefer having distinct function calls, but not necessarily distinct object properties.

**Comment by amadeus** [comment]
I have a fix for this in #838
